src/js/components/bulkDownload/awards/filters/dateRange/TimePeriodButtons.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * TimePeriodButtons.jsx
+ * Created by Lizzie Salita 11/6/17
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as fiscalYearHelper from 'helpers/fiscalYearHelper';
+import { awardDownloadOptions } from 'dataMapping/bulkDownload/bulkDownloadOptions';
+
+import FiscalYearButton from './buttons/FiscalYearButton';
+import DateRangeButton from './buttons/DateRangeButton';
+
+const propTypes = {
+    currentStartDate: PropTypes.string,
+    currentEndDate: PropTypes.string,
+    handleDateChange: PropTypes.func
+};
+
+export default class TimePeriodButtons extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.generateFYButtons = this.generateFYButtons.bind(this);
+        this.generateDateRangeButtons = this.generateDateRangeButtons.bind(this);
+    }
+
+    generateFYButtons(col) {
```

Comment:
This is approach is going to result in there always being two columns of FY. But as the available years increase, this column will grow taller while the other two columns with the today/this year/etc options will stay the same height.

Reply:
Spoke with @geoffgeiling and he wants to show just the 10 most recent years, so I set `earliestFY` to `currentFY - 10`